### PR TITLE
AppVeyor: Use libgtk+ from msys2

### LINKED ISSUE #234: Use msys2 for the AppVeyor build
I'm opening this to ask if it would be appreciated if I would change appveyor.yml to use msys2 to do the builds.
Advantages:
 - The [official guide](http://www.gtk.org/download/windows.php) to build on Windows recommends using msys2
 - Probably simpler build script

Disadvantages:
 - Only one GTK version supported

### PR DESCRIPTION
**Pull request by rnestler**
This is only to test if the build would work with this configuration.
It would fix #234

### CODE REVIEW

**Comment by gkoz** [comment]
What can I say, it does seem to work :)

**Comment by GuillaumeGomez** [comment]
@gkoz: So what do you say? Do you want we merge it or not?

**Comment by gkoz** [comment]
Okay, I guess I'm fine with having just this mode in appveyor. I'd like to avoid unnecessary verbosity though:

File: appveyor.yml [hunk 1]
```diff
@@ -1,59 +1,19 @@
...
-      }
-  - cmd: SET PATH=%MINGW%\bin;C:\Deps\gtk\bin;C:\Rust\bin;%PATH%
-
-cache:
-  C:\Deps
+  - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-${env:RUST_VERSION}-${env:TARGET}.exe"
+  - rust-%RUST_VERSION%-%TARGET%.exe /VERYSILENT /NORESTART /COMPONENTS="Rustc,Cargo,Std" /DIR="C:\Rust"
+  - SET PATH=C:\Rust\bin;C:\Program Files\Git\cmd;C:\Program Files\Git\usr\bin
+  - SET PATH=C:\msys64\mingw%BITS%\bin;C:\msys64\usr\bin;%PATH%
+  - echo %PATH%
```
Comment (gkoz) [inline appveyor.yml]: I don't think we need this

File: appveyor.yml [hunk 2]
```diff
@@ -61,7 +21,6 @@ build_script:
   - echo paths = ["."] > .cargo\config
   - git clone -q --depth 50 -b pending https://github.com/gtk-rs/examples _examples
   - cd _examples
-  - "echo Features: %FEATURES%"
-  - cargo build --features "%FEATURES%"
+  - cargo build --verbose
```
Comment (gkoz) [inline appveyor.yml]: The `echo` and `--verbose` are not necessary now but the `--features gtk_3_14` is.

File: appveyor.yml (hunk 1 above)
Comment (rnestler) [inline appveyor.yml]: The `echo %PATH%`? Yeah I can remove that.

File: appveyor.yml (hunk 2 above)
Comment (rnestler) [inline appveyor.yml]: OK

**Comment by gkoz** [comment]
Thanks, looks good :+1:

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**